packages/host/app/services/matrix-service.ts

```diff
@@ -509,6 +529,66 @@ export default class MatrixService extends Service {
     }
   }
 
+  private async initSlidingSync() {
+    let accountData = await this.client.getAccountDataFromServer<{
+      realms: string[];
+    }>(APP_BOXEL_REALMS_EVENT_TYPE);
+
+    let lists: Map<string, MSC3575List> = new Map();
+    lists.set(SLIDING_SYNC_AI_ROOM_LIST_NAME, {
+      ranges: [[0, SLIDING_SYNC_LIST_RANGE_END]],
+      filters: {
+        is_dm: false,
```

Comment:
Currently, this is adequate, since we only have AI rooms and Auth rooms, but generally we have been considering a room an AI room if the aibot it one if it's members. Is it possible to use that criteria here?

Reply:
Unfortunately, the filter options are very limited, as you can see [here](https://github.com/matrix-org/matrix-spec-proposals/blob/kegan/sync-v3/proposals/3575-sync.md?plain=1#L418-L478).